prime-router/src/main/kotlin/DocumentationFactory.kt

```diff
@@ -33,7 +33,20 @@ object DocumentationFactory {
         appendLabelAndData(sb, "Name", displayName)
         appendLabelAndData(sb, "Type", element.type?.name)
         appendLabelAndData(sb, "PII", if (element.pii == true) "Yes" else "No")
-        appendLabelAndData(sb, "Format", csvField?.format)
+
+        if (element.type?.name == "CODE") {
+            when (csvField?.format) {
+                "\$display" ->
```

Comment:
just to make it obvious that `$display` and `$alt` are treated the same use `"\$display", "\$alt"` so it matches both in a single case

Reply:
Fixed.  Thanks.